src/components/buttons.js

```diff
@@ -26,6 +26,7 @@ import { SvgIcon } from './SvgIcon';
  * @typedef IconButtonBaseProps
  * @prop {string} icon - Icon is required for icon buttons
  * @prop {string} title - Title is required for icon buttons
+ * @prop {never} [children] - children are not allowed (use LabeledButton instead)
```

Comment:
This is so handy! I had been wondering how to "remove" a property from a typedef and hadn't been able to google this up.

Reply:
I was recently reminded about `never` watching this video: https://www.youtube.com/watch?v=vXh4PFwZFGI